openwisp_radius/api/free_radius/views.py

```diff
@@ -0,0 +1,230 @@
+import logging
```

Comment:
let's simplify: please move the code to `openwisp_radius/api/freeradius_views.py` instead of `openwisp_radius/api/free_radius/views.py`.

Better to have both view files at the same level.